client/verta/verta/_registry/modelversion.py

```diff
@@ -318,6 +250,9 @@ def log_artifact(self, key, artifact, overwrite=False, _extension=None):
             Whether to allow overwriting an existing artifact with key `key`.
 
         """
+        self._log_artifact(key, artifact, _CommonCommonService.ArtifactTypeEnum.BLOB, overwrite=overwrite)
+
+    def _log_artifact(self, key, artifact, artifact_type, _extension=None, method=None, overwrite=False):
```

Comment:
I know this'll make the diff messier, but let's remove that underscore since this entire method is now private.
```suggestion
    def _log_artifact(self, key, artifact, artifact_type, extension=None, method=None, overwrite=False):
```

Reply:
Fixed.